geode-core/src/main/java/org/apache/geode/pdx/JSONFormatter.java

```diff
@@ -181,6 +204,10 @@ public static String toJSON(PdxInstance pdxInstance) {
    * @throws JSONFormatterException if unable to create the JSON document
    */
   public static byte[] toJSONByteArray(PdxInstance pdxInstance) {
+    return new JSONFormatter().toJsonByteArray(pdxInstance);
+  }
+
+  public byte[] toJsonByteArray(PdxInstance pdxInstance) {
```

Comment:
Having two methods in a public API with the same spelling & only casing differences is a little odd.  Also, this needs a javadoc.